apps/dashboard/lib/rclone_util.rb

```diff
@@ -128,11 +128,20 @@ def touch(remote, path)
     end
 
     def mkdir(remote, path)
+      path = Pathname.new(path)
       full_path = remote_path(remote, path)
       o, e, s = rclone("mkdir", full_path)
       if e.include?("Warning: running mkdir on a remote which can't have empty directories does nothing")
-        # TODO: Could most likely do some kind of workaround here, e.g. rclone touch remote:path/.somefile
-        raise RcloneError.new(s.exitstatus), "Remote does not support empty directories"
+        # Workaround for remotes that don't support empty directories.
+        # Touch a .directory file in the directory that is being created
+        begin
+          touch(remote, path.join(".directory"))
+          if !directory?(remote, path)
+            raise StandardError
+          end
+        rescue => e # Actual error messages from touch and directory? are not relevant, raise new error
+          raise RcloneError.new(s.exitstatus), "Remote does not support empty directories"
```

Comment:
This PR may have to wait for the other one as it has similar update here that you may need to pickup.

Reply:
Internationalized it, will resolve the conflicts in either of the PRs when needed.